docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst

```diff
@@ -163,7 +163,7 @@ To prevent unexpected behavior, this feature enforces a unique package name and
 uploaded to a participant node.
 This closes a loophole where the participant node allowed multiple DARs with
 the same package name and version. For backward compatibility, this
-restriction only applies for packages compiled with LF >= 1.16. If LF <
+restriction only applies for packages compiled with LF >= 1.17. If LF <
 1.15 is used, there can be several packages with the same name and
```

Comment:
The following line probably needs updating to be "if LF < 1.16", or a note needs to be made of the behaviour in 1.16

Reply:
As discussed offline, everything is as if 1.16 had never existed, so it's fine for its behavior to be unspecified.